src/lib/use-layout.ts

```diff
@@ -1,23 +1,28 @@
 import React from 'react';
 import RN from 'react-native';
+import { useIsKeyboardShown } from './use-is-keyboard-shown';
 
 interface Layout {
   width: number;
   height: number;
 }
 
-export default function (): [
-  Layout,
-  (event: RN.LayoutChangeEvent) => void | undefined,
-] {
+export default function (
+  keyboardShouldNotSetHeight = false,
+): [Layout, (event: RN.LayoutChangeEvent) => void | undefined] {
   const [layout, setLayout] = React.useState<Layout>({
     width: 0,
     height: 0,
   });
 
+  const isKeyboardVisible = useIsKeyboardShown();
+
+  const shouldSetHeight =
+    keyboardShouldNotSetHeight && isKeyboardVisible ? false : true;
```

Comment:
I'd maybe make it a bit more readable: `!(keyboardShouldNotSetHeight && isKeyboardVisible)`

Reply:
Yeah I agree, will change this